examples/hello-world/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "useDefineForClassFields": true,
+    "module": "ESNext",
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
```

Comment:
I think technically not DOM, since this example runs in node?

Reply:
Yeah ngl I just copied these from the vite project so it's defo wrong